lumigator/python/mzai/backend/backend/tests/conftest.py

```diff
@@ -107,40 +111,79 @@ def db_session(db_engine: Engine):
             session.rollback()
 
 
-@pytest.fixture(scope="session")
-def localstack_container():
-    """Initialize a LocalStack test container."""
-    with LocalStackContainer("localstack/localstack:3.4.0") as localstack:
-        yield localstack
+# @pytest.fixture(scope="session")
+# def localstack_container():
+#     """Initialize a LocalStack test container."""
+#     with LocalStackContainer("localstack/localstack:3.4.0") as localstack:
+#         yield localstack
 
 
-@pytest.fixture(scope="session", autouse=True)
-def setup_aws(localstack_container: LocalStackContainer):
-    """Setup env vars/AWS resources for use with the LocalStack container."""
+@pytest.fixture(scope="function")
+def fake_s3_storage() -> dict:
+    return dict()
+
+@pytest.fixture(scope="function")
```

Comment:
in the future we should be able to simplify this by removing a direct S3 client call and passing info into the S3FS mock directly https://github.com/mozilla-ai/lumigator/issues/473